sculture-android/app/src/main/res/layout/tag_description.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename this layout to activity_tag.xml. If we don't use naming convention on layout directory, the directory will become mess.